app/scripts.babel/contentscript.js

```diff
@@ -127,14 +129,15 @@ function toggle_epic_cards() {
 
 function hide_epics() {
   chrome.storage.sync.get('epicsToHide', function(items) {
-    let epic_collection = items.epicsToHide.split(',');
-
-    window.v1hiddenEpics = [];
-
-    epic_collection.forEach(function(number) {
-      number = number.trim();
-      hide_epic(`Epic:${number}`);
-    });
+    if (typeof items.epicsToHide != 'undefined') {
```

Comment:
Similar to comment above, rather than checking for undefined here, I think it's better to make the first argument to `.get` be an object that gives a fallback value of empty string for `epicsToHide`.